.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+      - migrate-github-actions
```

Comment:
Add this line to run github actions by definition file of this pull request.
I'll delete this line before merge this pull request.